buildci.sh

```diff
@@ -73,7 +73,7 @@ if [[ ! -z "$CHANGED_DIR" ]] && [[ -e "bundles/$CHANGED_DIR" ]]; then
 else
     echo "Build all"
     echo "MAVEN_OPTS='-Xms1g -Xmx2g -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'" > ~/.mavenrc
-    mvnp clean install -B -DskipChecks=true -DskipTests=true
+    mvnp clean install -B -DskipChecks=true
```

Comment:
I fear we are not yet ready for this :(

Reply:
And it takes to long to build with tests doesn't it?

Reply:
> And it takes to long to build with tests doesn't it?

No I believe it justs fits but when using both checks and tests it becomes critical

Reply:
done